protobuf/src/main/java/module-info.java

```diff
@@ -98,6 +98,26 @@
     requires static com.github.spotbugs.annotations;
     requires static java.annotation;
 
+    // only open protoc to com.google.protobuf
+    opens com.hedera.hapi.platform.state.legacy to
+            com.google.protobuf;
     opens org.hiero.block.api.protoc to
             com.google.protobuf;
+    opens org.hiero.block.internal.protoc to
+            com.google.protobuf;
+    opens com.hedera.services.stream.proto to
+            com.google.protobuf;
+    opens com.hederahashgraph.api.proto.java to
+            com.google.protobuf;
+    opens com.hederahashgraph.service.proto.java to
+            com.google.protobuf;
+    opens com.hedera.hapi.block.stream.protoc to
+            com.google.protobuf;
+    opens com.hedera.hapi.block.stream.input.protoc to
+            com.google.protobuf;
+    opens com.hedera.hapi.block.stream.output.protoc to
+            com.google.protobuf;
+    opens com.hedera.hapi.platform.event.legacy to
+            com.google.protobuf;
+
 }
```

Comment:
should we not just open it in general? 

```suggestion
                // open for reflection commonly used by com.google.protobuf
    opens com.hedera.hapi.platform.state.legacy;
    opens org.hiero.block.api.protoc;
    opens org.hiero.block.internal.protoc;
    opens com.hedera.services.stream.proto;
    opens com.hederahashgraph.api.proto.java;
    opens com.hederahashgraph.service.proto.java;
    opens com.hedera.hapi.block.stream.protoc;
    opens com.hedera.hapi.block.stream.input.protoc;
    opens com.hedera.hapi.block.stream.output.protoc;
    opens com.hedera.hapi.platform.event.legacy;

}
```

Reply:
We _just_ restricted these classes.
the protoc compiled classes are toxic to code that's trying to use PBJ, and we don't want them in the build.
The proper resolution for these is to split the code that uses protoc (simulator and E2E suites) from the main build and have _only_ protoc used for that build while _only_ PBJ is used for the main build.